docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md

```diff
@@ -33,16 +35,24 @@ The following images must be available in your air-gapped environment:
   - `web-modeler-ee/modeler-webapp`
   - `web-modeler-ee/modeler-websockets`
 
+We currently have a script in the [camunda-helm-respository](https://github.com/camunda/camunda-platform-helm/blob/c6a6e0c327f2acb8746802fbe03b3774b8284de3/scripts/download-chart-docker-images.sh) that will assist in pulling and saving Docker images.
+
 ## Required Helm charts
 
 The following charts must be available in your air-gapped environment:
 
-- [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm)
-- [Elasticsearch Helm chart](https://github.com/elastic/helm-charts/tree/main/elasticsearch)
+- [Camunda Helm chart](https://artifacthub.io/packages/helm/camunda/camunda-platform)
```

Comment:
It's a good idea to use the chart from Artifacthub.io. Let's do that also for Elasticsearch and Keycloak.

Reply:
Added artifacthub.io links to Elasticsearch and Keycloak.

Reply:
Also PSQL and Bitnami common.

https://artifacthub.io/packages/helm/bitnami/postgresql
https://artifacthub.io/packages/helm/bitnami/common

Reply:
also added PSQL and Bitnami common artifacthub links.